src/App.js

```diff
@@ -1,15 +1,25 @@
 import React from 'react'
-import Header from './components/Header/Header'
+import { Route, Switch } from 'react-router-dom'
 import styled from 'styled-components'
-import WorkspaceList from './components/WorkspaceList/WorkspaceList'
 import workspaces from './assets/workspaces.json'
+import Header from './components/Header/Header'
+import WorkspaceDetail from './components/WorkspaceDetail/WorkspaceDetail'
+import WorkspaceList from './components/WorkspaceList/WorkspaceList'
 
 function App() {
   return (
     <AppGrid>
-      <Header />
+      <Header title="digitalnatur" />
       <Main>
-        <WorkspaceList workspaces={workspaces} />
+        <Switch>
+          <Route exact path="/">
+            <WorkspaceList workspaces={workspaces} />
+          </Route>
+          <Route
+            path="/workspace/:id"
+            children={<WorkspaceDetail workspaces={workspaces} />}
```

Comment:
In Zeile 16 steht das child-Element direkt in dem Route-Tag und hier als Children-Prop. Vielleicht solltest du dich für eine Variante entscheiden, der Konsistenz zuliebe 😊

Reply:
Danke für den Hinweis, das war mir bis jetzt gar nicht bewusst :-)